airgen/src/lib.rs

```diff
@@ -302,4 +334,79 @@ impl<'a> ASMPILConverter<'a> {
             is_permutation,
         }
     }
+
+    // Process machine parameters.
+    // Allows machines passed as argument to be referenced.
+    // TODO: support some elementary pil types as a machine parameter?
+    fn handle_parameters(&mut self, params: MachineParams, values: &Vec<Expression>) {
```

Comment:
```suggestion
    fn handle_parameters(&mut self, MachineParams(params): MachineParams, values: &Vec<Expression>) {
```
(to avoid the `.0`)?